frontend/public/locales/en/translation.json

```diff
@@ -329,6 +329,7 @@
     "comparisonPage": "Comparison page",
     "rateLater": "Rate-later list",
     "recommendationsPage": "Recommendations page",
+    "customizeYourDefaultSearchFilter": "Customize <1>the default search filters</1> according to your own preferences. Those filters are applied <4>only</4> when you access the recommendations from the left <6>side bar</6>.",
```

Comment:
Maybe "main menu" would be clearer for mobile users? As you prefer.

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/1581/commits/96048af3d463996ada0073864c2d7a5c72cb00a9